magpie-run

```diff
@@ -1042,6 +1042,165 @@ else
 fi
 
 #
+# Setup Phoenix
+#
+
+if [ "${PHOENIX_SETUP}" == "yes" ] && [ "${HBASE_SETUP}" == "yes" ] && [ "${prior_setup_successful}" == "true" ]
```

Comment:
I don't think we need to check for hbase_setup, b/c our magpie-check-inputs already checked for it being set to yes?